packages/transaction-controller/src/types.ts

```diff
@@ -170,6 +170,11 @@ export type TransactionMeta = {
    */
   estimateUsed?: string;
 
+  /**
+   * The fiat value of the transaction to be used to passed metrics.
+   */
```

Comment:
Can we be explicit about the expected format here as mentioned in the other comment?